.github/workflows/tag-release.yml

```diff
@@ -0,0 +1,41 @@
+name: Tag Release
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - 'main'
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true && github.head_ref == 'release'
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+
+      - name: Get version
+        id: get-version
+        shell: bash
+        run: |
+          VERSION=$(grep -Eo '([0-9]+\.*)+' ${{ vars.VERSION_FILE_PATH }})
+          echo "current-version=$VERSION" >> $GITHUB_ENV
+
+      - name: Generate token
+        uses: tibdex/github-app-token@v1
+        id: generate-token
+        with:
+          app_id: ${{ vars.FRONTEND_PROJECT_BOT_APP_ID }}
+          private_key: ${{ secrets.FRONTEND_PROJECT_BOT_TOKEN }}
```

Comment:
I'm not totally sure about this; following the changes to the release workflow

Reply:
This is a github bot that allows automation with actions to use a shared token instead of individual users using their tokens.